Add user profile

jingjingwang opened the pull request.

jingjingwang commented:
Also solves #1.

marlonkeating commented on civictechprojects/templates/project_signup.html:
```diff
@@ -69,32 +69,32 @@ <h2 class="form-group subheader">DETAILS</h2>
             <div class="form-group">
                 <label for="brief_description">Describe This Project</label>
-                <textarea class="form-control" id="brief_description" placeholder="This will appear as project introduction" rows=3></textarea>
+                <textarea class="form-control" name="brief_description" placeholder="This will appear as project introduction" rows=3></textarea>
```
This wasn't added by your change, but I noticed it during testing.  Name should be "project_description" in order for it to be saved by the view.

jingjingwang commented:
Added a few fixes. One thing that I'm not entirely sure is that, should we use an email address as a username. Few places would need to be revisited if one day we changed the design.

marlonkeating commented:
User profile design has changed, so closing this review.
